src/gamemodes/Experimental.js

```diff
@@ -38,13 +37,11 @@ class Experimental extends FFA {
         };
         Entity.MotherCell.prototype.onRemove = function () {
             self.nodesMother.removeUnsorted(this);
+            self.spawnMotherCell(server);
```

Comment:
@JMcB17
What are the functions called by `self.spawnMotherCell`? Well `server.safeSpawn` is called.
 Can `server.safeSpawn` fail to spawn somthing? Yes.
 That means that the mothercell count would slowly drop until there are none left.